clippy_lints/src/utils/attrs.rs

```diff
@@ -21,25 +21,27 @@ pub const BUILTIN_ATTRIBUTES: &[(&str, DeprecationStatus)] = &[
         DeprecationStatus::Replaced("cognitive_complexity"),
     ),
     ("dump", DeprecationStatus::None),
+    ("msrv", DeprecationStatus::None),
 ];
 
-pub struct LimitStack {
-    stack: Vec<u64>,
+#[derive(Default, Clone)]
+pub struct LimitStack<T> {
+    stack: Vec<T>,
 }
 
-impl Drop for LimitStack {
+impl<T> Drop for LimitStack<T> {
     fn drop(&mut self) {
-        assert_eq!(self.stack.len(), 1);
```

Comment:
Not really sure what to change here but this was causing a panic while running tests